error-response.go

```diff
@@ -104,6 +106,41 @@ func BodyToErrorResponse(errBody io.Reader) error {
 	return errorResponse
 }
 
+// ErrEntityTooLarge input size is larger than supported maximum.
+var ErrEntityTooLarge = func(totalSize int64, bucketName, objectName string) error {
```

Comment:
Rewrite it as 
func ErrEntityTooLarge(totalSize int64, bucketName, objectName string) error {

same goes for all other errors.